### PR TITLE
Added BlueRepairSDK

### PR DESCRIPTION
**Pull request by arsenius7**
Relies on the fixed version of fetchGitExternal hook (CMakeLists.txt in subdirectory).

### CODE REVIEW

**Comment by adevress** [comment]
ok to test

File: bbp/nse/blue_repair_sdk/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{ fetchgitExternal, stdenv, boost, cmake, pkgconfig, gsl, bbpsdk, hdf5-cpp, zlib }:
+
+stdenv.mkDerivation rec {
+  name = "BlueRepairSDK-${version}";
```
Comment (adevress) [inline bbp/nse/blue_repair_sdk/default.nix]: Same for the name here. 

If you could switch it to something without camelcase to keep it consistent with other packages.

Comment (arsenius7) [inline bbp/nse/blue_repair_sdk/default.nix]: Looks like display-only name for me.
Does the naming convention matter here?

File: bbp/default.nix [hunk 2]
```diff
@@ -209,9 +209,10 @@ let
         bluejittersdk = callPackage ./nse/bluejittersdk {
         };
 
-
         bluepy = callPackage ./nse/bluepy {
+        };
 
+        blue_repair_sdk = callPackage ./nse/blue_repair_sdk {
```
Comment (arsenius7) [inline bbp/default.nix]: sure

**Comment by adevress** [comment]
It seems that Jenkins had a node problem...

**Comment by adevress** [comment]
retest this please

File: bbp/nse/blue_repair_sdk/default.nix (hunk 1 above)
Comment (adevress) [inline bbp/nse/blue_repair_sdk/default.nix]: It does not matter much. I just try usually to keep them consistent between variable name and package name. but this is your package, keep it like that if you want :)

**Comment by arsenius7** [comment]
I guess I have not made myself clear.
For this to work properly, fetchGitExternal hook should be fixed somewhat like you did on my PC (cd $out/subdir).
Would you like me to do it or you would prefer to modify it yourself?

**Comment by adevress** [comment]
@arsenius7  If you have already the modification in your local bbp-nixpkgs, feel free to jsut do a commit and a separated pull request. That would make us gain time.

**Comment by adevress** [comment]
@arsenius7  To make it part of the modules installed and make Jenkins build it, You also need to add it in the bbp/modules section as a new component and add it to the "all" list.

You can copy what Mike did for MUK here if you want a model https://github.com/BlueBrain/bbp-nixpkgs/pull/282

**Comment by arsenius7** [comment]
For testing this change I just hard-coded the path in fetchGitExternal.
I will propose a 'general' fix in another PR.

**Comment by adevress** [comment]
retest this please

**Pull request merged**